kuma/core/urlresolvers.py

```diff
@@ -134,6 +135,14 @@ def get_language(self):
 
         return settings.LANGUAGE_CODE
 
+    def get_chosen_language(self):
+        """If the request has a cookie set for language, return that language."""
+        language = self.request.COOKIES.get(settings.LANGUAGE_COOKIE_NAME)
+
+        # Need to check if his cookie language is supported by Kuma.
+        if language and language in dict(settings.LANGUAGES):
+            return language
```

Comment:
Should the default return be the empty string ``""``, to match the return of ``split_path`` for no language?

Reply:
Also, not covered by tests, according to codecov